fastlane/Fastfile

```diff
@@ -16,6 +16,19 @@ lane :test_autocreation do
   )
 end
 
+lane :test_autocreation_without_prompts do
```

Comment:
just curious, if it did ask for prompts how does the test fail?

Reply:
In code we have additional checks `Helper.test?`, which returns `true` in test env. So all prompts skipped, e.g.:
```rb
if Helper.test? || should_create_app || UI.confirm("App with name #{app_name} not found, create one?")
```